### PR TITLE
allow '|' characters in identifiers (that's the default delimiter for…

### PR DESCRIPTION
**Pull request by bcail**
… TemplateHTTPResolver

### CODE REVIEW

**Comment by alexwlchan** [comment]
Resolves #378.

File: tests/webapp_t.py [hunk 1]
```diff
@@ -228,6 +228,22 @@ def test_many_slash_info_request(self):
...
+        #image request
+        path = u'/%s/full/full/0/default.jpg' % identifier
+        req = _get_werkzeug_request(path)
+        loris_request = webapp.LorisRequest(req, False, None)
+        self.assertEqual(loris_request.request_type, u'image')
+        self.assertEqual(loris_request.ident, encoded_identifier)
+        #info request
+        path = u'/%s/info.json' % identifier
+        req = _get_werkzeug_request(path)
+        loris_request = webapp.LorisRequest(req, False, None)
```
Comment (alexwlchan) [inline tests/webapp_t.py]: Minor: please could you use keyword arguments here (and on L237) so it’s clearer what these values mean? i.e.

```python
webapp.LorisRequest(req, redirect_id_slash_to_info=False, proxy_path=None)
```

(Should we consider defaulting those values?)

**Review by alexwlchan — APPROVED** [review]
👍 

One comment, but otherwise LGTM. Merge once you’ve added the keyword arguments.

File: tests/webapp_t.py (hunk 1 above)
Comment (bcail) [reply tests/webapp_t.py]: added keyword arguments, and then made default values and removed the arguments from the test completely.

**Review by alexwlchan — APPROVED** [review]
👌

**Pull request merged**